content/applications/marketing/marketing_automation/getting_started/first_campaign.rst

```diff
@@ -1,59 +1,70 @@
-====================
-Marketing Automation
-====================
+===============
+Getting started
+===============
 
-The Odoo Marketing Automation app automates a variety of marketing tasks by combining specific
+The Odoo *Marketing Automation* app automates a variety of marketing tasks, by combining specific
 rules and filters to generate timed actions. Instead of manually having to build each stage of a
-campaign (such as a series of timed massmails), the Marketing Automation app allows marketers to
-build the entire campaign, and all of its stages, in one place on one dashboard.
+campaign (such as a series of timed massmails), the *Marketing Automation* app allows marketers to
+build the entire campaign, and all of its stages, in one place on a single dashboard.
 
 Create a campaign
 =================
 
-To create a new automated marketing campaign, open the :guilabel:`Marketing Automation` app and
-click :guilabel:`Create`. On the :guilabel:`Campaign` page, there are the following smart buttons
-and fields:
+To create a new automated marketing campaign, open the :menuselection:`Marketing Automation` app,
+and click the :guilabel:`Create` button in the upper-left corner. Doing so reveals a blank marketing
+automation campaign detail form on a separate page.
+
+On the blank marketing automation campaign page, the following smart buttons and fields are
+available:
 
 .. image:: first_campaign/marketing-template-sample.png
    :align: center
    :alt: A dashboard showing the creation of a new marketing automation campaign in Odoo.
 
+**Smart buttons**
+
 - :guilabel:`Templates`: represents the number of pre-configured mail templates being used in this
-  particular campaign. (Templates can always be created on-the-fly as well).
+  particular campaign. (Templates can always be created on-the-fly, as well).
```

Comment:
A sentence that's in entirely in parenthesis is not grammatically correct.
Avoid idioms like "on-the-fly"

```suggestion
  particular campaign. (Templates can always be created on-the-fly, as well).
```